### DISCUSSION TITLE
[Esame 15 gennaio 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#453] [Esami M-Z]
![15 gen es.4](https://github.com/user-attachments/assets/026bd18e-c0e0-4354-8692-ecc0b0c138be)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/bfa61db0-cd71-4a10-8b34-1722d1fb4d4e" alt="soluzione"  />

</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/8117217a-7be6-499f-b461-4e722166d723)

**Comment by niccolo-fato** [comment]
<img width="1280" height="782" alt="image" src="https://github.com/user-attachments/assets/b8724dbf-8092-4621-bfd0-8f702f0dc12a" />
